### PR TITLE
remove git merge tags

### PR DESCRIPTION
**Pull request by barrydegraaff**

### CODE REVIEW

**Comment by barrydegraaff** [comment]
@silentsakky a no brainer again

**Comment by barrydegraaff** [comment]
@Prashantsurana I am waiting for this fix, is looks somebody did a git merge and a manual conflict resolution. And forgot to remove some tags.

File: WebRoot/templates/prefs/Pages.template [hunk 1]
```diff
@@ -2128,11 +2128,8 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
 		<tr>
 			<td><div><$=ZmMsg.mobileConfigurationDownloadList$></div></td>
 			<td><div id='${id}_configType' tabindex=0></div></td>
-<<<<<<< HEAD
 			<td><div id='${id}_downloadButton' tabindex=0></div></td>
```
Comment (silentsakky) [inline WebRoot/templates/prefs/Pages.template]: There shouldn't be two download buttons

Comment (barrydegraaff) [reply WebRoot/templates/prefs/Pages.template]: I only see one button in the UI, what line are we talking about here?

Comment (barrydegraaff) [reply WebRoot/templates/prefs/Pages.template]: Wait see them, hold on.

Comment (silentsakky) [reply WebRoot/templates/prefs/Pages.template]: Fixed in https://github.com/Zimbra/zm-web-client/pull/479

**Comment by silentsakky** [comment]
Fixed in https://github.com/Zimbra/zm-web-client/pull/479

**Pull request closed without merging**